packages/editor/src/components/entities-saved-states/index.js

```diff
@@ -160,16 +130,13 @@ export function EntitiesSavedStatesExtensible( {
 			</Flex>
 
 			<div className="entities-saved-states__text-prompt">
-				<div
-					className="entities-saved-states__text-prompt--header-wrapper"
-					id={ renderDialog ? dialogLabel : undefined }
```

Comment:
removing this id will make the dialog labeling fail

Reply:
yes but I would argue that given we need to pass down the unique ID, how this would be better than using the current approach that passed down the `renderDialog` prop? The only difference would be that the logic is abstracted in a wrapper component but I'm not sure that would be a substantial improvement code-wise.

Reply:
True, but looking at the original code, it seems that the aria tags were missing anyway. I’m not entirely sure how much impact the aria tags would have, but we could consider #67351

Reply:
True, they were missing because the `renderDialog` prop was not passe down correctly. Looking at the original PR  that introduced thie behavior https://github.com/WordPress/gutenberg/pull/59622, the intent is to make all 'Save panels' behave like a sort of modal dialog and use the `useDialog` props and also the IDs for aria-labelledby and aria-describedby.